tests/cpp/common/test_json.cc

```diff
@@ -143,9 +143,26 @@ TEST(Json, TestParseObject) {
 }
 
 TEST(Json, ParseNumber) {
-  std::string str = "31.8892";
-  auto json = Json::Load(StringView{str.c_str(), str.size()});
-  ASSERT_NEAR(get<JsonNumber>(json), 31.8892f, kRtEps);
+  {
```

Comment:
Take a look at https://github.com/dmlc/dmlc-core/blob/808f485387f9a03f78fa9f1159f387d0d91b7a28/test/unittest/unittest_param.cc#L18 and see if we should adopt some of the test cases.